potpie/integrations/integrations/logging.py

```
@@ -0,0 +1,63 @@
+    def critical(self, msg, *args, **kwargs) -> None:
+        self._log(logging.CRITICAL, msg, *args, **kwargs)
+
+    fatal = critical
+
+    def isEnabledFor(self, level: int) -> bool:  # noqa: N802
+        return self._logger.isEnabledFor(level)
+
+    def getChild(self, suffix: str) -> "StructuredLogger":  # noqa: N802
+        return StructuredLogger(self._logger.getChild(suffix), self._bound_fields)
+
+    def _log(self, level: int, msg, *args, **kwargs) -> None:
```

Ruff (0.15.18)

[warning] Missing type annotation for `*args`

(ANN002)